packages/demo/.eslintrc.js

```diff
@@ -12,6 +12,7 @@
         "project": [
           "tsconfig.json"
         ],
+        "tsconfigRootDir": __dirname,
```

Comment:
It was working in CI without this, but not on local. This is the recommended way to do it (and probably the only way): https://typescript-eslint.io/linting/typed-linting/monorepos#one-tsconfigjson-per-package-and-an-optional-one-in-the-root